docs: Edge Function docs (#18738)
supabase/supabase: apps/docs/pages/guides/functions/local-development.mdx

export const meta = {
  id: 'functions-local-development',
  title: 'Local development',
  description: 'Setup local development environment for Edge Functions.',
  subtitle: 'Setup local development environment for Edge Functions.',
}

We recommend installing the Deno CLI and related tools for local development.

## Deno Support

You can follow the [Deno guide](https://deno.com/manual@v1.32.5/getting_started/setup_your_environment) for setting up your development environment with your favorite editor/IDE.

## Deno with Visual Studio Code

When using VSCode, you should install both the Deno CLI and the the Deno language server [via this link](vscode:extension/denoland.vscode-deno) or by browsing the extensions in vscode and choosing to install the _Deno_ extension.

## Deno support in subfolders

> In a given workspace (or workspace folder), sub-paths can be enabled for Deno, while code outside those paths will be not be enabled and the vscode built-in JavaScript/TypeScript language server will be used.

Reviewer: ```suggestion
You can enable the Deno language server for specific sub-paths in a workspace, while using VSCode's built-in JavaScript/TypeScript language server for all other files.
```
